_observing-your-data/query-insights/top-n-queries.md

```diff
@@ -67,16 +72,63 @@ PUT _cluster/settings
 
 ## Monitoring the top N queries 
 
-You can use the Insights API endpoint to obtain top N queries by latency:
+You can use the Insights API endpoint to obtain top N queries for all metric types:
 
 ```json
 GET /_insights/top_queries
 ```
 {% include copy-curl.html %}
 
-Specify a metric type to filter the response by metric type (latency is the only supported type as of 2.12):
+Specify a metric type to filter the response:
 
 ```json
 GET /_insights/top_queries?type=latency
 ```
-{% include copy-curl.html %}
\ No newline at end of file
+{% include copy-curl.html %}
+
+```json
+GET /_insights/top_queries?type=cpu
+```
+{% include copy-curl.html %}
+
+```json
+GET /_insights/top_queries?type=memory
+```
+{% include copy-curl.html %}
+
+## Exporting top N query data
+
+You can configure your desired exporter to export top N query data to different sinks, allowing for better monitoring and analysis of your OpenSearch queries. Currently, the supported exporters are:
+- [Debug exporter](#configuring-a-debug-exporter)
+- [Local index exporter](#configuring-a-local-index-exporter)
+
+### Configuring a debug exporter
+
+To configure a debug exporter, update the exporter setting for the desired metric type. For example, to export the top N queries by latency using the debug exporter, send the following request:
```

Comment:
`exporter` setting?

Reply:
This is not a full setting name so I'll leave without code font.